### ISSUE TITLE
Right hand editor width is 1px less when multiple editors are showing

### ISSUE THREAD

**Issue by Eldaw** [Microsoft/vscode#31309]
- VSCode Version: Code 1.14.2 (cb82febafda0c8c199b9201ad274e25d9a76874e, 2017-07-19T23:34:09.706Z)
- OS Version: Windows_NT ia32 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
spellright|ban|1.0.27
python|don|0.6.9
cpptools|ms-|0.12.1
csharp|ms-|1.11.0
PowerShell|ms-|1.4.1
sort-lines|Tyr|1.3.0
---

If I have the same file open in a 2-pane view, with the vertical splitter exactly centered (I've even double-clicked it to confirm it's in the center), then the editor in each pane has a different maximum line length. Please see the attached screenshot for an example. Here, I've enabled word wrapping, and I've got the same file open in both editors. Notice that on the left side I've got the letter 'B' as the last character on the first line and then the letter 'C' wraps to the next line. But on the right side the letters 'B' and 'C' both wrap to the next line. In this example, the editor on the left can fit 71 characters, while the editor on the right can only fit 70 characters.
![vscode_line_lengths](https://user-images.githubusercontent.com/3789849/28514868-8268b3f0-7053-11e7-943f-efba9a5a6cfb.png)

Reproduces without extensions: Yes/No

**Comment by bpasero** [comment]
I cannot reproduce on macOS. @alexandrudima can you try on Windows? maybe this is a rounding decimals issue where the right hand editor is getting 1px more.

**Comment by Eldaw** [comment]
@bpasero I'm not exactly sure how to see the size in pixels for each individual editor with the developer tools, but when I open the developer tools the size of the editors is reduced anyway which messes with the particular example I was highlighting in the screenshots. I'm pretty sure you're correct that it is a rounding decimals issue, as the problem is only evident at certain window sizes. For example, in my screenshots where I highlighted the problem, the dimensions of the overall window were 1400x1000. Now that I've resized the overall application window to 1564x1000, the problem goes away (i.e. I get exactly the same number of characters in each pane).

I guess it would be worthwhile to make sure that even if you do have a situation where one pane has 1 or 2 more pixels, that the wrapping is always done in such a way that the same number of characters is displayed per line so that you avoid a situation where you're getting 2 different views of a document even when the splitter is in the middle.

**Comment by Eldaw** [comment]
@bpasero This is a bug report. Not a feature request.

**Comment by bpasero** [comment]
@Eldaw sorry, that was not intentional (the message). I wanted to close this issue because it lacks more information (e.g. how to reproduce). I just tried on Windows and no matter how I resize the Window, the editor wraps correctly if the 2 editor sizes are identical. That is why I asked you to find out what the size of each editor is to see if it is the same or not:

![image](https://user-images.githubusercontent.com/900690/33238073-71e67ece-d285-11e7-99f2-0f13b18d0ced.png)

**Comment by bpasero** [comment]
@Eldaw agreed that was a bit too quick. 

I can reproduce on Windows and it looks like even though both editor containers have the same width, the right one gets 1px less width due to the fact that there is a 1px border between them. That seems like enough to produce the different wrapping.

**Comment by bpasero** [comment]
@Eldaw do you still see this?

**Comment by Eldaw** [comment]
@bpasero For some time now I've just been using my VS Code window sized to exactly 1564x1000 pixels, which avoided the problem described in this bug report (where the overall window size was set to 1400x1000). I've just done a test now where I've resized the window to 1400x1000 and I don't see the issue now. I haven't tested at any other sizes.

Previously you'd said you were able to reproduce it too. Are you able to reproduce it now? Maybe it has been fixed because another similar issue I reported has been fixed: https://github.com/Microsoft/vscode/issues/31312
